tests/dnsrecord/env_cleanup.yml

```diff
@@ -1,135 +1,144 @@
 ---
```

Comment:
The patch to this file is bigger because indentation was wrong and yamllint complained about it.